dipper/sources/GeneOntology.py

```diff
@@ -371,8 +371,10 @@ def process_gaf(self, gaffile, limit, id_map=None):
                         else:
                             model.addSynonym(gene_id, syn)
 
-                for txid in taxon.split('|'):
-                    tax_curie = re.sub(r'taxon:', 'NCBITaxon:', txid)
+                # First taxon is for the gene, after the pipe are interacting taxa
+                tax_curie = taxon.split('|')[0].replace('taxon', 'NCBITaxon')
+                # this is a required field but good to safe
+                if tax_curie:
                     geno.addTaxon(tax_curie, gene_id)
```

Comment:
might be interesting to check if exists and if so,  is it different?
also might be unnecessarily slow

Reply:
if it's different from the expected taxon for the file?